src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2672,23 +2672,14 @@
   <data name="CloseSelectedTabDescription" xml:space="preserve">
     <value>Close current tab</value>
   </data>
-  <data name="OpenNewPaneDescription" xml:space="preserve">
-    <value>Open new pane</value>
-  </data>
   <data name="CloseActivePaneDescription" xml:space="preserve">
     <value>Close active pane</value>
   </data>
-  <data name="FocusLeftPane" xml:space="preserve">
-    <value>Focus left pane</value>
-  </data>
-  <data name="FocusLeftPaneDescription" xml:space="preserve">
-    <value>Switch focus to the left pane</value>
+  <data name="FocusOtherPane" xml:space="preserve">
+    <value>Focus other pane</value>
```

Comment:
```suggestion
    <value>Focus the other pane</value>
```

Reply:
The title is supposed to be short so I think we can leave as is.